packages/flutter/lib/src/services/system_chrome.dart

```diff
@@ -85,21 +85,89 @@ class SystemChrome {
   /// embedder (if any). The embedder may choose to ignore unsupported
   /// overlays.
   ///
+  /// This method will schedule the embedder update to be run in a microtask.
+  /// Any subsequent calls to this method during the current event loop will
+  /// overwrite the pending value to be set on the embedder.
+  ///
   /// Arguments:
```

Comment:
btw we normally avoid using such a formal style with "Arguments:" and "Return Value:" in the dartdocs, in favour of just multiple paragraphs of prose that convey the same information. That way if it doesn't quite fit the format well, it's not a problem.

not saying you have to change this in this PR though.